README.md

```diff
@@ -6,4 +6,35 @@
 
 # imap-flow
 
-A thin abstraction layer handling IMAP client and server flows.
+The `imap-flow` repository provides a thin abstraction layer that handles IMAP client and server flows.
+One of the main features of `imap-flow` is that it abstracts away literal handling, making it easier to work with IMAP.
+
+## Playground
+
+This repository also serves as a playground for crates built on `imap-flow`.
+These will eventually be moved into their own repositories.
+
+Notably, we have the `proxy`, `tasks`, and `tag-generator` workspace members.
+
+* `proxy` is an already usable (but still not production-ready) IMAP proxy.
+  It gracefully forwards unsolicited responses, abstracts away literal processing, and `Debug`-prints messages.
+  Proxies are great for challenging the usability of a library, and we use them to validate our design decisions.
+  (See [./proxy/README.md].)
+* `tasks` is our prototype of a higher-level IMAP library that abstracts away command and response handling into `Task`s.
+  This crate will eventually become what a client or server implementor should use to get IMAP right.
+* `tag-generator` generates process-wide unique (and unguessable) IMAP tags.
+  This crate is here for organizational reasons and may be moved (or inlined) eventually.
+
+# License
+
+This crate is dual-licensed under Apache 2.0 and MIT terms.
```

Comment:
We havn't talked about a license yet. Are we on the same page? :-)

Reply:
I agree.